lib/cli/src/generators/ANGULAR/index.js

```diff
@@ -14,6 +14,7 @@ import {
   writeFileAsJson,
   copyTemplate,
 } from '../../helpers';
+import { supportedStoryFormats } from '../../project_types';
```

Comment:
`STORY_FORMAT`?

Reply:
I thought of defining it like STORY_FORMAT, but to keep consistency I would need to refactor also:
projectTypes
supportedLanguages
supportedFrameworks
supportedTemplates
notInstallableProjectTypes
installableProjectTypes

Which I'm fine with if you think that's the best way to go!

Reply:
I think these string constants should be capitalized. The object constants I’m fine with camel case. But it’s not a big deal to me, which is why I approved the PR as is

Reply:
Cool! Please check the last 4 commits I just pushed